metrics-core/src/macros.rs

```diff
@@ -119,14 +122,16 @@ macro_rules! gauge {
     };
 }
 
+// TODO: we may need another macro for recording non time-interval data into
+// summary types. Use case would be for inserting bucketized counts into the
+// metrics library.
+
```

Comment:
Unless I'm misunderstanding you here I think this is the `value!` macro?

Reply:
You're right. =)